docs/adrs/003-store-room-allocation-in-memory.md

```diff
@@ -0,0 +1,84 @@
+---
+status: accepted
+date: 2024-05-21
+deciders: Kamil Kołodziej, Jakub Pisarek, Radosław Szuma
+consulted: Michał Śledź
+informed: Fishjam Team
+---
+# Route incoming traffic within the cluster without a need to provide room_id
+
+## Context and Problem Statement
+
+In the current architecture after we create a room in cluster we return on which instance (Jellyfish host) it was created.
+This means that Jellyfish client needs to make request to only that specific instance.
+Which effectively leads to the situation where client may need to keep mapping between room_id and jellyfish host.
+
+We would like to hide information about internal architecture of the cluster and the exact JF hosts as well as make it easier for clients so they can use only cluster endpoint.
+
+## Decision Drivers
+
+* Cluster architecture should be an internal information
+* `room_id` should be a sufficient information to route the requests
+
+## Considered Options
+
+* Store room allocation across the cluster using in-memory ETS
+* Store room allocation across the cluster using database
+* Leave it as it is and update the SKDs
```

Comment:
Shouldn't we at least mention the 'include information about the node holding the room in room ID' option?